schemas/dex/SCHEMA.md

```diff
@@ -0,0 +1,186 @@
+|-----------------------|----------------------------------------------------------------------|--------|
+| timestamp             | The timestamp of the transaction.                                    | number |
+| chain_id              | The standard id of the chain.                                        | number |
+| block_number          | The block number of the trade.                                       | number |
+| log_index             | The log index of the event recorded.                                 | number |
+| transaction_hash      | The hash of the transaction.                                         | string |
+| user_address          | The address that initiates the transaction (ie, the transaction signer). | string |
+| contract_address      | The smart contract address the user interacted with (ie, transaction to). | string |
+| taker_address         | The taker, the address that receives the output of the swap (ie, could be the same as user_address unless a proxy contract/router is used). | string |
+| pool_address          | The contract address of the pool.                                    | string |
+| token0_address        | The contract address of token0.                                      | string |
+| token0_amount         | The amount of token0.                                                | number |
+| token1_address        | The contract address of token1.                                      | string |
+| token1_amount         | The amount of token1.                                                | number |
+| burn_amount           | The amount of LP tokens burned, decimal normalized.                  | number |
+| burn_amount_usd       | (Optional) The amount of the burn in USD.                                       | number |
+
+###  V2 Syncs
+| Property              | Description                                                          | Type   |
+|-----------------------|----------------------------------------------------------------------|--------|
+| timestamp             | The timestamp of the transaction.                                    | number |
+| chain_id              | The standard id of the chain.                                        | number |
+| block_number          | The block number of the trade.                                       | number |
+| log_index             | The log index of the event recorded.                                 | number |
+| transaction_hash      | The hash of the transaction.                                         | string |
+| user_address          | The address that initiates the transaction (ie, the transaction signer). | string |
+| contract_address      | The smart contract address the user interacted with (ie, transaction to). | string |
+| taker_address         | The taker, the address that receives the output of the swap (ie, could be the same as user_address unless a proxy contract/router is used). | string |
```

Comment:
we can probably remove these

the purpose of the sync event is just to get the snapshot update of the token balances

intiator,contract_address interacted are not useful here and might be hassle to get
there is no taker_address if user add/remove liquidity